examples/08-rust-kernel.rs

```diff
@@ -0,0 +1,74 @@
+use std::{path::PathBuf, array};
+
+use cudarc::{
+    driver::{CudaDevice, DriverError, LaunchAsync, LaunchConfig},
+    nvrtc::PtxCrate,
+};
+
+use std::time::Instant;
+
+fn main() -> Result<(), DriverError> {
+    let dev = CudaDevice::new(0)?;
+
+    // use compile_crate_to_ptx to build kernels in pure Rust
+    // uses experimental ABI_PTX
+    let kernel_path: PathBuf = "examples/rust-kernel/src/lib.rs".into();
+    let kernels = PtxCrate::compile_crate_to_ptx(&kernel_path).unwrap();
```

Comment:
Can you include the type of this variable? Just so its more clear what the type is 😄

Reply:
Added docstrings for everything.